src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -537,5 +531,38 @@ private static Set<File> toggleFiles(File folder, boolean enable) throws IOExcep
 
 		return changed;
 	}
-	
+
+	private static void notifyOperators(String message, CommandSender sender, RedirectingLogHandler logHandler) {
+		if (SkriptConfig.sendReloadingInfoToOps.value()) {
+			Bukkit.getOnlinePlayers().stream()
+				.filter(player -> player.isOp() && player.hasPermission("skript.reloadnotify") && !player.equals(sender))
+				.forEach(player -> {
+					if (logHandler.numErrors() == 0) {
+						Skript.info(player, message);
+					} else {
+						for (String errorMsg : logHandler.getErrors()) {
+							SkriptLogger.sendFormatted(player, errorMsg);
+						}
+						Skript.error(player, message);
+					}
+				});
+
+			// Send the message to the console if the boolean is false
```

Comment:
this comment indicates that the brackets for the enclosing `if` are wrong

Reply:
this comment i accidentally left, it was in an earlier point of this PR, will remove